docs/rpc.md

```diff
@@ -284,7 +284,27 @@ state has all its values got from MPT with the specified stateroot. This allows
 to track the contract storage scheme using the specified past chain state. These
 methods may be useful for debugging purposes.
 
-#### `submitnotaryrequest` call
+#### P2PNotary extensions
+
+To utilize the following P2PNotary extensions, the P2P signature extensions
+should be enabled.
```

Comment:
One day P2P notary service will be moved out of extensions, thus, let's use something like "The following P2PNotary extensions can be used on P2P Notary enabled networks only". And `P2P signature extensions` term is only from NeoGo universe, in C# it can be something else (not implemented yet).

Reply:
Fixed.